reporting-cli/src/download-helpers.js

```diff
@@ -0,0 +1,336 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import puppeteer from 'puppeteer';
+import fs from 'fs';
+import { FORMAT, REPORT_TYPE, SELECTOR, AUTH, URL_SOURCE } from './constants.js';
+import { exit } from "process";
+import ora from 'ora';
+
+const spinner = ora();
+
+export async function downloadReport(url, format, width, height, filename, authType, username, password, tenant) {
+  spinner.start('Connecting to url ' + url);
+  try {
+    const browser = await puppeteer.launch({
+      headless: true,
+      args: [
+        '--no-sandbox',
+        '--disable-setuid-sandbox',
+        '--disable-gpu',
+        '--no-zygote',
+        '--font-render-hinting=none',
+        '--enable-features=NetworkService',
+        '--ignore-certificate-errors',
+      ],
+      executablePath: process.env.CHROMIUM_PATH,
```

Comment:
is CHROMIUM_PATH still needed? If so this also needs to be documented

Reply:
I kept CHROMIUM_PATH just in case anyone wants to use custom chromium on linux arm. Updated the doc.